controllers/actions.github.com/resourcebuilder.go

```diff
@@ -320,6 +320,9 @@ func mergeListenerPodWithTemplate(pod *corev1.Pod, tmpl *corev1.PodTemplateSpec)
 	// apply metadata
 	for k, v := range tmpl.Annotations {
 		if _, ok := pod.Annotations[k]; !ok {
```

Comment:
Good catch! It was also good to learn that `_, ok := pod.Annotations[k]` doesn't fail when `pod.Annotations` is nil. Thanks!